modules/serverless-traffic-management-kn.adoc

```diff
@@ -0,0 +1,60 @@
+[id="serverless-traffic-management-kn_{context}"]
+= Splitting traffic between revisions by using the Knative CLI
+
+You can use the `kn service update` command to split traffic between revisions of a service.
+
+.Example command
+[source,terminal]
+----
+$ kn service update <service_name> --traffic <revision>=<percent>
+----
+
+Where:
+
+* `<service_name>` is the name of the Knative service that you are configuring traffic routing for.
+* `<revision>` is the revision that you want to configure to receive a percentage of traffic. You can either specify the name of the revision, or a tag that has been applied to the revision by using the `--tag` flag.
+* `<percent>` is the percentage of traffic that you want to send to the specified revision.
+
+[IMPORTANT]
+====
+The `--traffic` flag can be specified multiple times in one command, and is valid only if the sum of the `Percent` values in all flags totals 100.
+====
+
+All traffic-related flags can be specified using a single `kn service update` command. `kn` defines the precedence of these flags. The order of the flags specified when using the command is not taken into account.
+
+The precedence of the flags as they are evaluated by `kn` are:
+
+. `--untag`: All the referenced revisions with this flag are removed from the traffic block.
+. `--tag`: Revisions are tagged as specified in the traffic block.
+. `--traffic`: The referenced revisions are assigned a portion of the traffic split.
+
+[id="serverless-traffic-management-kn-tags"]
+= Using tags
+
+You can add tags to revisions and then split traffic according to the tags you have set:
```

Comment:
This file seems to be a catalog of command examples--extremely valuable, btw.  But it's the catalog nature that makes me think you want this as a reference module.

Maybe add a heading for each of these examples:

"== Example: Route blue and green traffic to different revisions"

The key consideration is that there's no "-ing" suffix to deceive readers into believing it's a procedure.

Then you have other examples like...

"== Example: Split traffic 80/20 to different revisions"

(Those two ideas for titles are not awesome.)